independent-projects/tools/devtools-common/src/main/java/io/quarkus/devtools/commands/handlers/ListCategoriesCommandHandler.java

```diff
@@ -20,6 +25,7 @@ public class ListCategoriesCommandHandler implements QuarkusCommandHandler {
     private static final String ID_FORMAT = "%-20s";
     private static final String CONCISE_FORMAT = "%-50s %-50s";
     private static final String FULL_FORMAT = "%-30s %-20s %s";
+    private ObjectMapper objectMapper = new ObjectMapper();
```

Comment:
I agree with @gsmet here, I am bit reluctant to instantiate a new ObjectMapper here. I believe we have already done that somewhere else in the code

Reply:
Sure. Where ? Or tell me how I can get an instance?

Reply:
I removed the usage of json and just use Object, so no need anymore to go to json